File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    string sOdd = "";
    string sEven = "";
    for(int i = 0; i < num_list.size(); i++)
    {

        if(num_list[i] % 2 == 0)
        {
            sEven += to_string(num_list[i]);
        }
        else 
        {
            sOdd += to_string(num_list[i]); 
        }
    }
    answer = stoi(sEven) + stoi(sOdd);
    return answer;
}

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        string sOdd = "";
        string sEven = "";
        for(int i = 0; i < num_list.Length; i++)
        {

            if(num_list[i] % 2 == 0)
            {
                sEven += num_list[i].ToString();
            }
            else 
            {
                sOdd += num_list[i].ToString(); 
            }
        }
        answer = Int32.Parse(sEven) + Int32.Parse(sOdd);
        
        return answer;
    }
}